### PR TITLE
`sync` -- log tweaks

### PR DESCRIPTION
**Pull request by danlaine**
## Why this should be merged

See comments

## How this works

n/a

## How this was tested

n/a

### CODE REVIEW

File: x/sync/manager.go [hunk 1]
```diff
@@ -145,6 +145,8 @@ func (m *Manager) Start(ctx context.Context) error {
 		return ErrAlreadyStarted
 	}
 
+	m.config.Log.Info("starting sync", zap.Stringer("target root", m.config.TargetRoot))
```
Comment (danlaine) [inline x/sync/manager.go:148]: will help for debugging/showing stuff to user

File: x/sync/manager.go [hunk 2]
```diff
@@ -539,14 +541,13 @@ func (m *Manager) Wait(ctx context.Context) error {
 
 	root, err := m.config.DB.GetMerkleRoot(ctx)
 	if err != nil {
-		m.config.Log.Info("completed with error", zap.Error(err))
```
Comment (danlaine) [inline x/sync/manager.go:542]: I feel like the caller should worry about logging this

File: x/sync/manager.go [hunk 3]
```diff
@@ -600,7 +604,7 @@ func (m *Manager) setError(err error) {
 	m.errLock.Lock()
 	defer m.errLock.Unlock()
 
-	m.config.Log.Error("syncing failed", zap.Error(err))
+	m.config.Log.Error("sync errored", zap.Error(err))
```
Comment (danlaine) [inline x/sync/manager.go:607]: `syncing` --> `sync` for parallelism

File: x/sync/manager.go [hunk 4]
```diff
@@ -662,7 +666,7 @@ func (m *Manager) completeWorkItem(ctx context.Context, work *workItem, largestH
 	}
 
 	// completed the range [work.start, lastKey], log and record in the completed work heap
-	m.config.Log.Info("completed range",
+	m.config.Log.Debug("completed range",
```
Comment (danlaine) [inline x/sync/manager.go:669]: I feel like the user doesn't need to see all these, so made it debug

File: x/sync/manager.go [hunk 5]
```diff
@@ -568,6 +571,7 @@ func (m *Manager) UpdateSyncTarget(syncTargetRoot ids.ID) error {
 		return nil
 	}
 
+	m.config.Log.Debug("updated sync target", zap.Stringer("target", syncTargetRoot))
```
Comment (danlaine) [inline x/sync/manager.go:574]: for debugging

**Review by darioush — APPROVED** [review]

**Pull request merged**